### PR TITLE
LPS-196001 | Test Fix | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Investigate failure in LocalFile.ABTesting#EditVariant

Hi! :)
This pr is corresponding to this ticket: [LPS-196001](https://liferay.atlassian.net/browse/LPS-196001)

**Explanation:**
This test failure occurred after a change in behavior in the A/B test. It is no longer possible to add more than one variant to the A/B test.
[LRAC-14353](https://liferay.atlassian.net/browse/LRAC-14353)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/abtest/ABTest.path [hunk 1]
```diff
@@ -142,7 +142,7 @@
 </tr>
 <tr>
 	<td>VARIANT_DROPDOWN</td>
-	<td>xpath=(//div[@class='autofit-col']//*[contains(@class, 'dropdown-toggle btn-monospaced btn btn-outline-borderless btn-outline-secondary')])[${key_index}]</td>
+	<td>//tbody//*[contains(@aria-label,"Show Actions")][${key_index}]</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/abtest/ABTest.path:145]: This one is already fixed by https://liferay.atlassian.net/browse/LPS-195990

**Review by Tim-Cao — CHANGES_REQUESTED** [review]

**Comment by Tim-Cao** [comment]
@rafaellajordao, would you mind refactoring? Thanks

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/13850 @rafaellajordao

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**